src/trackerhandle.py

```
@@ -86,27 +93,38 @@ def print_tracker_result(
+                if has_status_message and (print_messages or (print_links and not link_url)):
+                    result_parts.append(escape(Redaction.redact_private_info(status_message)))
+
+                message = f"{tracker_label} {' '.join(result_parts)}" if result_parts else None
+            else:
+                if "status_message" in status and "data error" in str(status["status_message"]):
+                    logger.info(f"[red]{tracker_label} {escape(str(status['status_message']))}[/red]")
+                    return
+
+            if message is not None:
                 logger.info(message, extra={"highlighter": None})
         except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)